platform/windows/test_gpt2_c.vcxproj

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
```

Comment:
Maybe we should remove all 32 build targets. Even the smallest GPT-2 124M will not work on x32

Reply:
@azret - yep, there's a lot of tweaks here that we can do. Let's get the minimal set in and then we'll optimize the project files/batch files/cmake files. This will NOT be part of the PR.